### PR TITLE
Improving vrfy plots and Fixing blank plots

### LINKED ISSUE #1126: Bkgerr Plots from Marine Verify Task are blank
- Since we fixed #1057 , taking `h` from its `output_data_file` instead of `input_grid_file` is not enough to plot both zonal and meridional figures.
- This fix will be added while working on #1005 
- There will be no task failures happening.

### PR DESCRIPTION
**Pull request by apchoiCMD**
#### This PR includes,

- Add contour lines for all plots
- Changes some colormaps
- Partially improved and will do more based on what we addressed #1005
- Resolved #1126

### CODE REVIEW

File: ush/soca/soca_vrfy.py [hunk 1]
```diff
@@ -95,14 +96,26 @@ def plotHorizontalSlice(config):
     bounds = config['bounds']
 
     fig, ax = plt.subplots(figsize=(8, 5), subplot_kw={'projection': projs[config['proj']]})
-    plt.pcolormesh(np.squeeze(grid.lon),
-                   np.squeeze(grid.lat),
-                   slice_data,
-                   vmin=bounds[0], vmax=bounds[1],
-                   transform=ccrs.PlateCarree(),
-                   cmap=config['colormap'])
+    pcolormesh = plt.pcolormesh(np.squeeze(grid.lon),
```
Comment (guillaumevernieres) [inline ush/soca/soca_vrfy.py]: remove the pcolormesh and use contourf below

**Comment by guillaumevernieres** [comment]
After a bit of figure browsing:
- [ ] Something is wrong with hi (ice thickness). It looks way too thick, check if it's a plotting issue ... Or a model issue :fearful: 
- [x] Adjust colorbar for hs (snow thickness)
- [x] salinity colormap looks good but colorbar needs improvement
- [x] We need to add zonal and meridional sections of the background
- [x] Scatter size for Satellites obs's plot
- ...

**Comment by apchoiCMD** [comment]
#### List that we have to look at-
- We need to able to click though the `html` what we want to see
- Colorbar issue for min and max
- Achieve `COMROT` for the future by tar them to HPSS

**Comment by apchoiCMD** [comment]
Latest plots are here
```
/scratch1/NCEPDEV/da/Mindo.Choi/workflow_06032024/global-workflow/sorc/gdas.cd/build/gdas/test/gw-ci/GFSv17-3DVAR-C384mx025/COMROOT/GFSv17-3DVAR-C384mx025/gdas.20210630/06/analysis/ocean/vrfy
```

**Comment by apchoiCMD** [comment]
`Markersize = 0` with eva will not generate the plots, it seems `1` is minimum size for `map_plots`

File: ush/soca/soca_vrfy.py [hunk 2]
```diff
@@ -17,12 +17,13 @@
...
 def plotConfig(grid_file=[],
                data_file=[],
+               layer_file=[],
                variable=[],
                PDY=os.getenv('PDY'),
                cyc=os.getenv('cyc'),
                exp=os.getenv('PSLOT'),
                levels=[],
-               bounds=[],
+               # bounds=[],
```
Comment (guillaumevernieres) [inline ush/soca/soca_vrfy.py]: just remove if it's not used anymore

File: ush/soca/soca_vrfy.py [hunk 3]
```diff
@@ -44,12 +45,13 @@ def plotConfig(grid_file=[],
...
     config['grid file'] = grid_file
     config['fields file'] = data_file
+    config['layer file'] = layer_file
     config['PDY'] = PDY
     config['cyc'] = cyc
     config['exp'] = exp
     config['levels'] = [1]
     config['colormap'] = colormap
-    config['bounds'] = bounds
+    # config['bounds'] = bounds
```
Comment (guillaumevernieres) [inline ush/soca/soca_vrfy.py]: same as a bove

**Review by guillaumevernieres — DISMISSED** [review]
Looks good, just a few minor comments. Is it still failing when running the ci?

**Comment by apchoiCMD** [comment]
> Looks good, just a few minor comments. Is it still failing when running the ci?

No, it shouldn't be commented out and some bounds also are added,, now doing a final check-ups for you mentioned. Will push modified one soon-

**Review by guillaumevernieres — APPROVED** [review]

**Pull request merged**